src/main/resources/templates/detect-sh.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-echo "Detect Shell Script ${SCRIPT_VERSION}"
-
```

Comment:
Was this echo pre-existing? If yes, I would leave it.

Reply:
Yes it was pre-existing, but the problem is that it is getting echoed 2 times which is not looking good. As per discussion with Mayank removing one of them. The second one is on line 87 'echo "Detect Shell Script ${SCRIPT_VERSION}"'